common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -167,6 +171,11 @@ public void onPageLoaded(final String url) {
                                 mWebView.loadUrl("javascript:" + javascriptToExecute[0].replace("%", "%25"));
                             }
                         }
+
+                        //clearcerts if url has certauth
```

Comment:
You can add more context for why we are clearing the certPreference here ( copy paste from your PR description here, or add the bug link)